include/wx/qt/radiobut.h

```diff
@@ -8,6 +8,7 @@
 #ifndef _WX_QT_RADIOBUT_H_
 #define _WX_QT_RADIOBUT_H_
 
+
```

Comment:
This should not be here

Reply:
Thanks, I have removed it in the latest commit.